go/cmd/berty/main.go

```diff
@@ -208,6 +211,20 @@ func main() {
 			cleanup := globalPreRun()
 			defer cleanup()
 
+			if !miniInMemory && datastorePath == cacheleveldown.InMemoryDirectory {
+				storagePath := configdir.New("Berty Technologies", "Berty")
```

Comment:
![Manfreds-iMac - Screen Shot 2020-06-19 at 18 01 01](https://user-images.githubusercontent.com/94029/85153788-e6f57900-b256-11ea-8b82-fe4f2b6f6b77.png)

it works but looks very strange on linux, I'm okay to merge like this but I prefer to let you choose if you want to make it differently or not